src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -347,4 +395,23 @@ export function MissingSamplesMessage(
             </div>
         </div>
     );
+}
+
+export function sortDataIntoQuartiles<D>(
```

Comment:
I think jstat has utilties for generating quartiles. could use it?

Reply:
I'm using the jstat library for calculating the quartile limits, but I didn't see a method for sorting data into those quartiles